File: csharp/config/store/EntityStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NHINDirect.Config.Store
{
    public enum EntityStatus : byte
    {
        New = 0,
        Enabled = 1,
        Disabled = 2,
        Deleted = 3
    }
}

File: java/config/config-store/src/main/java/org/nhindirect/config/store/EntityStatus.java
package org.nhindirect.config.store;

import java.util.ArrayList;
import java.util.List;

public enum EntityStatus {
	NEW, 
	ENABLED, 
	DISABLED ;
	
	public static List<String> getEntityStatusList() {
		ArrayList<String> result = new ArrayList<String>();
		EntityStatus[] stati = EntityStatus.values();
		for (int i = 0; i < stati.length; i++) {
			result.add(stati[i].toString());
		}
		return result;
	}
}
